# English Indices of Deprivation

The dataset analyzed in this notebook presents the survey results of the 2025 English Indices of Deprivation survey. This survey "measure[s] relative levels of deprivation in 33,755 small areas or neighbourhoods, called Lower-layer Super Output Areas (LSOAs), in England."

Retrieved From: https://deprivation.communities.gov.uk/download-all

The Government of the United Kingdom defines deprivation as "people’s unmet needs, a lack of access to opportunities and resources which we might expect in our society." REFERENCE

The English Indices of Deprivation survey measures levels of deprivation along 7 where each domain is weighted differently to determine the Index of Multiple Deprivation (IMD) for each LSOA:
- Income Deprivation (22.5)
- Employment Deprivation (22.5)
- Education, Skills and Training Deprivation (13.5)
- Health and Disability Deprivation (13.5)
- Crime Deprivation (9.3)
- Barriers to Housing and Services Deprivation (9.3)
- Living Environment Deprivation (9.3)

In [25]:
import pandas as pd
import seaborn as sns
pd.set_option('display.max_columns', None)

In [26]:
df = pd.read_csv('../data/raw_data/deprivation.csv')

In [27]:
df.info()

<class 'pandas.DataFrame'>
RangeIndex: 33755 entries, 0 to 33754
Data columns (total 56 columns):
 #   Column                                                                                              Non-Null Count  Dtype  
---  ------                                                                                              --------------  -----  
 0   LSOA code (2021)                                                                                    33755 non-null  str    
 1   LSOA name (2021)                                                                                    33755 non-null  str    
 2   Local Authority District code (2024)                                                                33755 non-null  str    
 3   Local Authority District name (2024)                                                                33755 non-null  str    
 4   Index of Multiple Deprivation (IMD) Score                                                           33755 non-null  float64

In [28]:
df.isna().value_counts()

LSOA code (2021)  LSOA name (2021)  Local Authority District code (2024)  Local Authority District name (2024)  Index of Multiple Deprivation (IMD) Score  Index of Multiple Deprivation (IMD) Rank (where 1 is most deprived)  Index of Multiple Deprivation (IMD) Decile (where 1 is most deprived 10% of LSOAs)  Income Score (rate)  Income Rank (where 1 is most deprived)  Income Decile (where 1 is most deprived 10% of LSOAs)  Employment Score (rate)  Employment Rank (where 1 is most deprived)  Employment Decile (where 1 is most deprived 10% of LSOAs)  Education, Skills and Training Score  Education, Skills and Training Rank (where 1 is most deprived)  Education, Skills and Training Decile (where 1 is most deprived 10% of LSOAs)  Health Deprivation and Disability Score  Health Deprivation and Disability Rank (where 1 is most deprived)  Health Deprivation and Disability Decile (where 1 is most deprived 10% of LSOAs)  Crime Score  Crime Rank (where 1 is most deprived)  Crime Decile (where 1 is 

In [29]:
df.head()

,LSOA code (2021),LSOA name (2021),Local Authority District code (2024),Local Authority District name (2024),Index of Multiple Deprivation (IMD) Score,Index of Multiple Deprivation (IMD) Rank (where 1 is most deprived),Index of Multiple Deprivation (IMD) Decile (where 1 is most deprived 10% of LSOAs),Income Score (rate),Income Rank (where 1 is most deprived),Income Decile (where 1 is most deprived 10% of LSOAs),Employment Score (rate),Employment Rank (where 1 is most deprived),Employment Decile (where 1 is most deprived 10% of LSOAs),"Education, Skills and Training Score","Education, Skills and Training Rank (where 1 is most deprived)","Education, Skills and Training Decile (where 1 is most deprived 10% of LSOAs)",Health Deprivation and Disability Score,Health Deprivation and Disability Rank (where 1 is most deprived),Health Deprivation and Disability Decile (where 1 is most deprived 10% of LSOAs),Crime Score,Crime Rank (where 1 is most deprived),Crime Decile (where 1 is most deprived 10% of LSOAs),Barriers to Housing and Services Score,Barriers to Housing and Services Rank (where 1 is most deprived),Barriers to Housing and Services Decile (where 1 is most deprived 10% of LSOAs),Living Environment Score,Living Environment Rank (where 1 is most deprived),Living Environment Decile (where 1 is most deprived 10% of LSOAs),Income Deprivation Affecting Children Index (IDACI) Score (rate),Income Deprivation Affecting Children Index (IDACI) Rank (where 1 is most deprived),Income Deprivation Affecting Children Index (IDACI) Decile (where 1 is most deprived 10% of LSOAs),Income Deprivation Affecting Older People (IDAOPI) Score (rate),Income Deprivation Affecting Older People (IDAOPI) Rank (where 1 is most deprived),Income Deprivation Affecting Older People (IDAOPI) Decile (where 1 is most deprived 10% of LSOAs),Children and Young People Sub-domain Score,Children and Young People Sub-domain Rank (where 1 is most deprived),Children and Young People Sub-domain Decile (where 1 is most deprived 10% of LSO,Adult Skills Sub-domain Score,Adult Skills Sub-domain Rank (where 1 is most deprived),Adult Skills Sub-domain Decile (where 1 is most deprived 10% of LSOAs),Geographical Barriers Sub-domain Score,Geographical Barriers Sub-domain Rank (where 1 is most deprived),Geographical Barriers Sub-domain Decile (where 1 is most deprived 10% of LSOAs),Wider Barriers Sub-domain Score,Wider Barriers Sub-domain Rank (where 1 is most deprived),Wider Barriers Sub-domain Decile (where 1 is most deprived 10% of LSOAs),Indoors Sub-domain Score,Indoors Sub-domain Rank (where 1 is most deprived),Indoors Sub-domain Decile (where 1 is most deprived 10% of LSOAs),Outdoors Sub-domain Score,Outdoors Sub-domain Rank (where 1 is most deprived),Outdoors Sub-domain Decile (where 1 is most deprived 10% of LSOAs),Total population: mid 2022,Dependent Children aged 0-15: mid 2022,Older population aged 60 and over: mid 2022,Working age population 18-66 (for use with Employment Deprivation Domain): mid 2022
0,E01000001,City of London 001A,E09000001,City of London,8.742,26525,8,0.013,33730,10,0.014,33708,10,0.004,33755,10,-1.771,33108,10,-2.220,33698,10,10.950,29220,9,69.345,244,1,0.039,33304,10,0.012,33721,10,-2.902,33752,10,0.030,33749,10,4.437,33560,10,0.688,12832,4,1.207,1105,1,1.414,1586,1,1795,149,520,1248
1,E01000002,City of London 001B,E09000001,City of London,4.722,31203,10,0.018,33669,10,0.010,33734,10,0.169,33672,10,-1.549,32574,10,-2.277,33712,10,6.703,32640,10,43.890,3702,2,0.076,31744,10,0.026,33118,10,-1.830,33266,10,0.032,33747,10,3.589,33632,10,-0.628,18721,6,0.355,9591,3,1.839,592,1,1671,81,387,1324
2,E01000003,City of London 001C,E09000001,City of London,9.250,25913,8,0.107,25167,8,0.064,26985,8,3.269,30273,9,-0.292,20719,7,-0.765,27325,9,9.735,30400,10,41.163,4540,2,0.250,19647,6,0.153,16506,5,-0.756,27140,9,0.113,31467,10,2.509,33722,10,0.358,14241,5,0.318,10175,4,1.679,903,1,1896,136,432,1469
3,E01000005,City of London 001E,E09000001,City of London,19.884,1

#### Filtering for Tower Hamlets Only

In [30]:
df_tower_hamlets = df[df['LSOA name (2021)'].str.contains('Tower Hamlets')]

In [31]:
df_tower_hamlets.shape

(169, 56)

In [32]:
df_tower_hamlets.head()

,LSOA code (2021),LSOA name (2021),Local Authority District code (2024),Local Authority District name (2024),Index of Multiple Deprivation (IMD) Score,Index of Multiple Deprivation (IMD) Rank (where 1 is most deprived),Index of Multiple Deprivation (IMD) Decile (where 1 is most deprived 10% of LSOAs),Income Score (rate),Income Rank (where 1 is most deprived),Income Decile (where 1 is most deprived 10% of LSOAs),Employment Score (rate),Employment Rank (where 1 is most deprived),Employment Decile (where 1 is most deprived 10% of LSOAs),"Education, Skills and Training Score","Education, Skills and Training Rank (where 1 is most deprived)","Education, Skills and Training Decile (where 1 is most deprived 10% of LSOAs)",Health Deprivation and Disability Score,Health Deprivation and Disability Rank (where 1 is most deprived),Health Deprivation and Disability Decile (where 1 is most deprived 10% of LSOAs),Crime Score,Crime Rank (where 1 is most deprived),Crime Decile (where 1 is most deprived 10% of LSOAs),Barriers to Housing and Services Score,Barriers to Housing and Services Rank (where 1 is most deprived),Barriers to Housing and Services Decile (where 1 is most deprived 10% of LSOAs),Living Environment Score,Living Environment Rank (where 1 is most deprived),Living Environment Decile (where 1 is most deprived 10% of LSOAs),Income Deprivation Affecting Children Index (IDACI) Score (rate),Income Deprivation Affecting Children Index (IDACI) Rank (where 1 is most deprived),Income Deprivation Affecting Children Index (IDACI) Decile (where 1 is most deprived 10% of LSOAs),Income Deprivation Affecting Older People (IDAOPI) Score (rate),Income Deprivation Affecting Older People (IDAOPI) Rank (where 1 is most deprived),Income Deprivation Affecting Older People (IDAOPI) Decile (where 1 is most deprived 10% of LSOAs),Children and Young People Sub-domain Score,Children and Young People Sub-domain Rank (where 1 is most deprived),Children and Young People Sub-domain Decile (where 1 is most deprived 10% of LSO,Adult Skills Sub-domain Score,Adult Skills Sub-domain Rank (where 1 is most deprived),Adult Skills Sub-domain Decile (where 1 is most deprived 10% of LSOAs),Geographical Barriers Sub-domain Score,Geographical Barriers Sub-domain Rank (where 1 is most deprived),Geographical Barriers Sub-domain Decile (where 1 is most deprived 10% of LSOAs),Wider Barriers Sub-domain Score,Wider Barriers Sub-domain Rank (where 1 is most deprived),Wider Barriers Sub-domain Decile (where 1 is most deprived 10% of LSOAs),Indoors Sub-domain Score,Indoors Sub-domain Rank (where 1 is most deprived),Indoors Sub-domain Decile (where 1 is most deprived 10% of LSOAs),Outdoors Sub-domain Score,Outdoors Sub-domain Rank (where 1 is most deprived),Outdoors Sub-domain Decile (where 1 is most deprived 10% of LSOAs),Total population: mid 2022,Dependent Children aged 0-15: mid 2022,Older population aged 60 and over: mid 2022,Working age population 18-66 (for use with Employment Deprivation Domain): mid 2022
3995,E01004198,Tower Hamlets 002B,E09000030,Tower Hamlets,45.952,3183,1,0.490,2736,1,0.254,3521,2,25.669,11023,4,1.032,4364,2,1.031,3853,2,39.225,2343,1,34.347,7176,3,0.609,4455,2,0.752,292,1,-0.076,17559,6,0.328,6530,2,4.407,33563,10,6.439,694,1,-0.206,20117,6,2.033,331,1,1521,271,228,1072
3996,E01004199,Tower Hamlets 002C,E09000030,Tower Hamlets,44.165,3553,2,0.541,1739,1,0.222,5208,2,27.729,10004,3,0.812,6339,2,0.805,5846,2,37.962,2774,1,34.365,7166,3,0.802,869,1,0.757,282,1,-0.272,20534,7,0.358,4560,2,4.085,33584,10,6.272,825,1,-0.317,22501,7,2.218,155,1,1775,361,204,1267
3997,E01004200,Tower Hamlets 005A,E09000030,Tower Hamlets,22.522,12777,4,0.255,12002,4,0.094,19975,6,4.046,29356,9,0.585,8697,3,0.373,11276,4,28.353,7836,3,45.454,3236,1,0.573,5523,2,0.983,12,1,-0.757,27167,9,0.135,29353,9,3.695,33619,10,4.775,2476,1,0.437,8396,3,1.792,666,1,1524,173,74,1295
3998,E01004201,Tower Hamlets 002D,E09000030,Tower Hamlets,25.579,10737,4,0.357,6918,3,0.135,13211,4,9.806,22876,7

Just like with the LSOA datasets there are indeed 169 LSOA codes within tower hamlets, which means that we can now proceed to join the datasets.

### Exporting Data

In [34]:
df_tower_hamlets.to_csv('../data/clean_data/deprivation_th.csv')